unittest/reachable-workspace.cpp

```diff
@@ -296,7 +296,7 @@ BOOST_AUTO_TEST_CASE(test_reachable_workspace)
 
   // Load the urdf model
   Model model;
-  pinocchio::buildModels::manipulator(model);
+  pinocchio::buildModels::manipulator(model, true);
```

Comment:
I'm not sure we should activate mimic here. Reachable workspace use integration that should already be tested.